tests/requirements-tools.txt

```diff
@@ -1,12 +1,12 @@
 # Work-around for a bug in execnet 1.4.1
-# :todo: remove this again, once a version of execnet fixing this is released
-hg+https://htgoebel@bitbucket.org/htgoebel/execnet
+git+https://github.com/pytest-dev/execnet.git
 
 # For running tests
-pytest >= 2.7.3  # Testing framework.
+git+https://github.com/xoviat/pytest.git@capture
```

Comment:
Neat -- so removing/not using pytest-catchlog avoids these errors? If so, that's great! Or did I misunderstand https://github.com/pytest-dev/pytest-xdist/issues/141?